js/models/chat/ChatMessage.js

```diff
@@ -66,10 +74,20 @@ export function processMessage(message) {
   processedMessage = $message.html();
 
   wordsToAnchorify.forEach(word => {
-    const w = word.startsWith('ob://') ? word.slice(5) : word;
+    let href = !is.url(word) && !word.startsWith('ob://') ?
+      `#ob://${word}` : word;
+
+    if (is.url(word)) {
+      const link = document.createElement('a');
+      link.setAttribute('href', word);
+      if (link.protocol === 'file:') {
+        href = `http://${word}`;
```

Comment:
If the protocol was file, wouldn't it only work locally?

Reply:
file is what the protocol comes up as when you don't give a protocol. For example if the user types `www.espn.com`, the protocol (using this method) comes up as `file` and that to us is an indicator that we need to add in the `http`.